### ISSUE TITLE
Can't use Jekyll 3.0 due to Ruby version

### ISSUE THREAD

**Issue by jbrains** [openshift-quickstart/jekyll-openshift#4]
I'm using this quickstart, but now I can't regenerate my Jekyll blog on my OpenShift server, because Jekyll 3.0 requires Ruby 2.0 and this quickstart uses Ruby 1.8.7-p374.

I want to be able to regenerate my Jekyll blog on OpenShift so that I can schedule blog posts in the future and let a cron job running on OpenShift regenerate my blog every hour. Right now, this doesn't work.

I'm trying to go back to an earlier version of Ruby for now, but I'd appreciate a fix. Either this quickstart installs Ruby 2.0 or higher *or* switches to the Ruby cartridge in order to serve up the static files of the blog itself. Or something else. I'm not picky, as long as it supports Jekyll 3.0 and higher.

(I would happily do this myself if I could figure out how to do it. I'm digging in and it's very slow-going.)

**Comment by jbrains** [comment]
After some hunting, I figured out that I can do the following to put Ruby 2.0 on my path, but is there a better way to tell this cartridge to use Ruby 2.0 to run bundler when regenerating the blog?

```
#!/bin/bash
echo "$(date) Regenerating blog." >> $OPENSHIFT_LOG_DIR/regenerating-blog.log
export LC_CTYPE=en_CA.UTF-8
export LANG=en_CA.UTF-8
export LD_LIBRARY_PATH=/opt/rh/mysql55/root/usr/lib64:/opt/rh/ror40/root/usr/lib64:/opt/rh/ruby200/root/usr/lib64
export PATH=$OPENSHIFT_REPO_DIR/libs:~/.gem/bin:/opt/rh/ruby200/root/usr/bin:$PATH
cd $OPENSHIFT_REPO_DIR && bundle install && bundle exec jekyll build
```

**Comment by bonzini** [comment]
I used this cartridge yesterday to generate a jekyll 3.3.0 blog. This was fixed by commit 0e79d8903727da0441010965a4071b8301adb5fc.

**Comment by jbrains** [comment]
Interesting that that commit happened 2015-10-29 but I had the problem on 2015-12-30. I would have to try it again and see what happens.

Thanks for the comment; it really helps.
